public/activity.html

```
@@ -190,405 +173,285 @@ <h3 class="font-bold text-lg mb-4 flex items-center">
-              <i class="fas fa-user-tie mr-2 text-teal-600 dark:text-teal-400"></i>Activity Host
+          <!-- List header -->
+          <div class="flex items-center justify-between">
+            <h2 class="text-lg font-bold text-gray-800 dark:text-gray-200 flex items-center gap-2">
+              <i class="fas fa-file-alt text-teal-600 dark:text-teal-400"></i>
+              Course Materials
+              <span id="mat-count-label" class="text-sm font-normal text-gray-500 dark:text-gray-400"></span>
             </h2>
-            <div id="host-info" class="flex items-center gap-4">
-              <!-- Populated by JS -->
-            </div>
+            <button onclick="loadMaterials()" title="Refresh"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)